app/README.md

```diff
@@ -172,10 +172,11 @@ npm version patch
 git push
 ```
 
-At this point it's a good idea to check that Travis confirms all tests are passing. Then,
+At this point it's a good idea to check that Travis confirms all tests are passing. Then, for example if the new version is '2.1.15', 
```

Comment:
Maybe add the note that `npm version patch` doesn't create the commit, it just updates the package files. So either add the lines:
```
git add package.json package-lock.json
git commit -m "Upgrade to 2.1.15"
```
or fix the `npm version` hook :)

Reply:
ah yep, good point, I was pushing with a UI in mind. I'll add those lines.

I took a look at https://docs.npmjs.com/cli/version#description .. it should just work since `--git-tag-version` flag is default true, setting it again doesn't seem to help. Not sure what's going on there.